src/features/filter.ts

```diff
@@ -1,10 +1,47 @@
-export const actions = { /* put action creators here */};
+type QueryFilterAction = {
+  type: 'queryFilter/SET';
+  payload: string;
+};
+
+const setQueryFilter = (query: string): QueryFilterAction => ({
+  type: 'queryFilter/SET',
```

Comment:
its better to use prefix for action types names same as reduser name or part of state name
fix in all cases
```suggestion
  type: 'filter/SET_QUERY',
```